packages/lib-react-components/src/SpacedHeading/SpacedHeading.js

```diff
@@ -2,10 +2,9 @@ import SpacedText from '../SpacedText'
 import { Heading } from 'grommet'
 import { node, number, object, oneOfType, string } from 'prop-types'
 import React from 'react'
-import styled from 'styled-components'
 
 function SpacedHeading(props) {
-  const { 
+  const {
```

Comment:
I'd remove this and destructure the function arguments directly. Your defaults should then appear in the storybook.